### PR TITLE
Bug 0012641 - Detailed Descendant Report

### PR DESCRIPTION
**Pull request by GaryGriffin**
Update to include Event media in report

### CODE REVIEW

File: DetDescendantReport-images/detdescendantreport.gpr.py
Comment (hgohel) [inline DetDescendantReport-images/detdescendantreport.gpr.py:5]: As far as I understand it, version numbers are incremented automatically when the addon is built for release: [reference this post](https://gramps.discourse.group/t/what-is-the-process-of-merging-pull-requests-in-the-add-ons-project/2400/7?u=codefarmer) by @prculley.

File: DetDescendantReport-images/detdescendantreporti.py
Comment (hgohel) [inline DetDescendantReport-images/detdescendantreporti.py:17]: With author history permanently recorded in GitHub (along with exact changes made by the author per commit), adding attribution here is superfluous.

File: DetDescendantReport-images/detdescendantreport.gpr.py
Comment (GaryGriffin) [reply DetDescendantReport-images/detdescendantreport.gpr.py:5]: jpschewe made a further update and bumped to 1.0.11 .
I was previously told to bump the version when I submit so I did. 

I am a novice github user. Are you saying I should remove the .gpr.py from the PR so that the version number is not updated and let the release process do it instead?

File: DetDescendantReport-images/detdescendantreporti.py
Comment (GaryGriffin) [reply DetDescendantReport-images/detdescendantreporti.py:17]: When I rewrote a plugin (GetGOV) years ago and did not update the copyright, I was told that I should. I was just following that previous advise.

File: DetDescendantReport-images/detdescendantreport.gpr.py
Comment (jpschewe) [reply DetDescendantReport-images/detdescendantreport.gpr.py:5]: @hgohel Let us know how you'd like the versions handled. I'm the author of this report. I have a pending PR that depends on this one and intend to create another PR after that with some more updates.

File: DetDescendantReport-images/detdescendantreporti.py
Comment (jpschewe) [reply DetDescendantReport-images/detdescendantreporti.py:17]: @hgohel  I too have been adding this copyright, please let us know what is the preference for updating this information (or not) so that the PR can be merged.

File: DetDescendantReport-images/detdescendantreport.gpr.py
Comment (hgohel) [reply DetDescendantReport-images/detdescendantreport.gpr.py:5]: @GaryGriffin, @jpschewe In the discussion I referenced, @prculley says, "Note that the release process bumps up the version number and this results in a commit,..." which _implies_ that version numbers do not need to be manually updated. I don't see guidance in the [Addons Development](https://www.gramps-project.org/wiki/index.php/Addons_development) wiki, so I created a thread asking for clarification on [Gramps Development forum on Discourse](https://gramps.discourse.group/t/should-an-addons-pr-include-version-number-update/2591).

BTW, my review comments are not marked as "change requested" in GitHub, so it should not block this PR from being committed.

File: DetDescendantReport-images/detdescendantreporti.py
Comment (hgohel) [reply DetDescendantReport-images/detdescendantreporti.py:17]: This comment is also not marked as change required - don't let this stop the PR from being committed. For clarification I [created a discussion thread](https://gramps.discourse.group/t/updating-copyright-authorship-and-reason-for-change-in-source-code-required/2592) so maintainers can help clarify expectations.

**Review by hgohel — CHANGES_REQUESTED** [review]
Based on discussion in the Development forum, the version number change in the gpr.py file should be backed out as the version is bumped up during the build and release process.

**Pull request merged**
